tutorials/text/text_props.py

```diff
@@ -204,14 +212,16 @@
 # Korean, or Japanese.
 #
 # To set the default font to be one that supports the code points you
-# need, prepend the font name to ``'font.family'`` or the desired alias
-# lists ::
+# need, prepend the font name to ``'font.family'`` (or the desired alias
+# lists, which is not recommended) ::
```

Comment:
I would leave out the "not recommended" unless you write another paragraph to explain the motivation of having generic families at all.

Reply:
Added another paragraph 👍🏼